survey form.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+	 <meta charset="utf-8">
+	 <title>Survey Form</title>
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+	 <link rel="stylesheet" href="surveyform.css">
+ </head>
+ <body>
+	 <h1 id="title">Microverse Survey Form</h1>
+	 <p id="description">The form aims to create the best student experience in Microverse</p>
+	  <div class="input-headings">
+		  <form id="survey-form">
+			  <label for="name" id="name-label">Name: &nbsp;</label><br>
+			  <input type="text" id="name" placeholder="Enter your name" required><br>
```

Comment:
You can use the tag <div> with a class that could wrap the whole part of survey-form instead of <form> and id